src/pynxtools/dataconverter/validation.py

```diff
@@ -328,13 +328,24 @@ def check_nxdata():
 
     def handle_group(node: NexusGroup, keys: Mapping[str, Any], prev_path: str):
         variants = get_variations_of(node, keys)
```

Comment:
If you would a short doc here also that would be great as well. Because, the name `handle_group` has really a broad meaning. Other than that I think all are good.

Reply:
In this case I'm opting against this. These are all just inline groups. Their purpose is already to make the code more readable by chunking it into smaller functions and nobody can call this functions from outside. Commenting all of them would make this function even longer + we have to see if we keep it this way or use a validation which works directly on the hdf nodes (I already started to work on this in #333)